pkg/restapi/v1/oidc4ci/controller.go

```diff
@@ -655,6 +700,45 @@ func (c *Controller) oidcPreAuthorizedCode(
 	return &validateResponse, nil
 }
 
+// OidcRegisterDynamicClient handles OIDC Dynamic Client Registration (POST /oidc/register).
+func (c *Controller) OidcRegisterDynamicClient(e echo.Context) error {
```

Comment:
that name is auto-generated, I agree that it would be nice to have the proper acronym here but this is what we have :(, btw it's consistent across controller (i.e. there are `OidcPushedAuthorizationRequest`, `OidcAuthorize`, etc,

Reply:
Oh OK, thanks for letting me know!